extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,286 @@
+{
+    "asset" : {
+        "generator" : "Khronos glTF Blender I/O v0.9.36",
```

Comment:
Probably best to remove `"generator"`. We don't want to accidentally endorse blender supporting this extension.